src/ledger/LiabilitiesTests.cpp

```diff
@@ -505,7 +513,7 @@ TEST_CASE("liabilities", "[ledger][liabilities]")
             REQUIRE(tf->getSellingLiabilities(lm) == 0);
             if (res)
             {
-                REQUIRE(tf->getTrustLine().ext.v() == 1);
+                REQUIRE(tf->getTrustLine().ext.v() == (deltaLiabilities != 0));
```

Comment:
while this is correct, I find the implicit conversion from `true` to `1` quite obscure when used like this.